artemis-chart/templates/monitor-deployment.yaml

```diff
@@ -26,7 +28,7 @@ spec:
         - mountPath: /pvc
           name: monitor-pvc
           subPath: configs
-        command: ['sh', '-c', 'cp -n -u /configmaps/logging.yaml /configmaps/mon-services.conf /pvc/']
+        command: ['sh', '-c', 'false | cp -i /configmaps/logging.yaml /configmaps/mon-services.conf /pvc/ 2> /dev/null']
```

Comment:
shouldn't also the changes regarding
```
        {{- range $key, $value := .Values.nodeSelector }}
        {{ $key }}: {{ $value }}
        {{- end }}
      {{ toYaml . | nindent 8 }}
      {{- end }}
```
applied here as in the other deployments?